src/components/Breadcrumb/Breadcrumb.module.css

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.breadcrumb {
+  /* TODO: these CSS variables are missing in DS */
```

Comment:
On Figma we have the semantic layer (--palette-background-neutral), but we miss the "component layer", where we should define each component token. It's a task we have in backlog, but it has low priority since it has low value. We can discuss this decision and start adding this layer!

Reply:
I'd open an issue to start this discussion and then we fix this TODO then, ok?

@fedepini can you open it yourself?

Reply:
Opened it: https://github.com/mia-platform/design-system/issues/497